src/components/TextPanel/TextPanel.tsx

```diff
@@ -38,6 +39,12 @@ export const TextPanel: React.FC<Props> = ({
    * Styles
    */
   const styles = useStyles2(getStyles);
+  const panelStyles = options.styles ? replaceVariables(options.styles) : '';
+
+  /**
+   * Injects styles into the global scope (need for dt-row-___ classes)
+   */
+  injectGlobal(panelStyles);
```

Comment:
`injectGlobal` is not a good idea and a breaking change. I don't think we need it.

Reply:
updated